apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -67,6 +67,21 @@ function dateAccessor(value: Maybe<string>, intl: IntlShape) {
   ) : null;
 }
 
+const fullNameCell = (pool: Pool, intl: IntlShape) => {
+  return (
+    <span>
+      {getFullNameHtml(pool.owner?.firstName, pool.owner?.lastName, intl)}
+    </span>
```

Comment:
Does this need to be wrapped in a `span`? If not then I think we could remove it to reduce the tiny amount of DOM bloat.

Reply:
it was to try and handle typing issues that kept popping up like

![image](https://user-images.githubusercontent.com/40485260/217312155-562489a7-7dc8-4c8b-8349-c67ece5a6171.png)

Reply:
okay, I assumed there was something like that going on here. Would be nice to not need a span but def not in scope of this issue :P